index.html

```diff
@@ -248,51 +186,89 @@ <h3><dfn>Sampling Frequency</dfn> and <dfn>Reporting Frequency</dfn></h3>
   </p>
 </section>
 
+<section> <h2>Pressure States</h2>
+  <p>
+    <dfn>Pressure states</dfn> represents the minimal set of useful states that allows web applications
+    to react to changes in compute pressure with minimal degration in quality or service, or user
+    experience.
+  </p>
+  <pre class="idl">
+    enum PressureState { "nominal", "fair", "serious", "critical" };
+  </pre>
+  <p>
+    The <dfn>PressureState</dfn> enum represents the [=pressure state=] with the following states:
+  </p>
+  <p>
+    <style>
+      @counter-style pressure-states-emoji {
+        system: cyclic;
+        symbols: "⚪" "🟢" "🟡" "🔴";
+        suffix: " ";
+      }
+      .pressure-states {
+        list-style-type: pressure-states-emoji;
+      }
+    </style>
+    <ul class="pressure-states">
+      <li>
+        <b>Nominal</b>: Work is minimal and the system is running on lower clock speed to preserve power.
+      </li>
+      <li>
+        <b>Fair</b>: The system is doing fine, everything is smooth and it can take on additional work without issues.
+      </li>
+      <li>
+        <b>Serious</b>: There is some serious pressure on the system, but it is sustainable and the system
+        is doing well, but it is getting close to its limits:
+        <ul>
+          <li>
+            Clock speed (depending on AC or DC power) is consistently high
+          </li>
+          <li>
+            Thermals are high but system can handle it
+          </li>
+        </ul>
+        At this point, if you add more work the system may move into critical.
+      </li>
+      <li>
+        <b>Critical</b>: The system is now about to reach its limits, but it hasn’t reached <em>the</em> limit yet.
+        Critical doesn’t mean that the system is being actively throttled, but this state is not sustainable for
+        the long run and might result in throttling if the workload remains the same. This signal is the last call
+        for the web application to lighten its workload.
+      </li>
+    </ul>
+  </p>
+</section>
+
 <section> <h2>Compute Pressure Observer</h2>
 The Compute Pressure Observer API enables developers to understand the utilization characteristics of a CPU.
 
 <section data-dfn-for="ComputePressureObserverCallback">
   <h3>The <a>ComputePressureUpdateCallback</a> callback</h3>
   <pre class="idl">
     callback ComputePressureUpdateCallback = undefined (
-      ComputePressureEntry update,
+      sequence&lt;ComputePressureEntry&gt; entries,
       ComputePressureObserver observer
     );
   </pre>
-  This callback will be invoked when [=CPU=] [=frequency=] and/or [=utilization=] crosses the user
-  set thresholds.
+  This callback will be invoked when the [=pressure state=] changes.
```

Comment:
Should you mention rate limitations or link to it from there?

Reply:
Good point! I figured out how :)

Reply:
But I'm not seeing any change ;(